vectorsearch_embedding/workload.json

```diff
@@ -0,0 +1,32 @@
+{% import "benchmark.helpers" as benchmark with context %}
+
+{
+  "version": 2,
+  "description": "MS MARCO is a collection of datasets focused on deep learning in search",
```

Comment:
can we use trec-covid dataset instead of MSMarco? MS marco is a big dataset and might take a long time to complete.

Reply:
Dataset is not available on Trec-covid homepage https://ir.nist.gov/covidSubmit/index.html (dataset link does not work), also there is not anything about a license or if commercial use is allowed.

I found the trec-covid dataset here https://github.com/beir-cellar/beir and here https://huggingface.co/datasets/BeIR/trec-covid, but no license there.

Do you know where is it available with license?

Reply:
if the dataset is available on BeIR we can use the dataset. in general the default dataset we should set for this should be a one which good enough in size that it can be completed in limited time. My experience with MSMARCO is as it around 8M dataset, the whole benchmarks only ingestion can take time. Hence I was suggesting a smaller dataset.

Reply:
@navneet1v - I see both MSMarco and Covid-trex available on BeIR: https://paperswithcode.com/dataset/beir

However, based on wbeckler and gkamat feedback on this PR: https://github.com/opensearch-project/opensearch-benchmark-workloads/pull/232#discussion_r1525618687, the license of the dataset cannot be 'non-commercial' .. Would you know how to verify covid-trex license information?

Reply:
@akashsha1 I don't have a way sorry.

Reply:
@navneet1v  - thanks. I did some research, and confirmed that covid-trec is part of the COVID19 Open Research dataset. 

The covid-trec research paper[0] links to the Open Research Dataset[1], which is licensed under Apache 2.0. This should comply with OpenSearch license.
We should be good to proceed with this license for your PR @vpehkone 

cc: @wbeckler , @gkamat 

[0]: https://arxiv.org/pdf/2005.04474v1.pdf
[1]: https://github.com/allenai/cord19/tree/master

Reply:
This is awesome. :) thanks @akashsha1 for providing the details.